django_pivot/utils.py

```diff
@@ -1,5 +1,8 @@
 from django.core.exceptions import FieldDoesNotExist
-from django.utils.encoding import force_text
+try:
+    from django.utils.encoding import force_str
+except ImportError:
+    from django.utils.encoding import force_text as force_str
```

Comment:
It looks like we won't hit the except block until force_str is actually removed, so even in Django 3.0 we will still be using force_str instead of force_text. I'm not sure how this affects the deprecation warning that you're seeing. What do you think about a version check here as well?